wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md

```diff
@@ -11,19 +11,311 @@ tags:
+- HardRock
+  1. [Rchetype - V Is For (Blue Dragon) [Insane]](https://osu.ppy.sh/beatmapsets/111914#osu/290919)
+  2. [Nero's Day At Disneyland - No Money Down, Low Monthly Payments (grumd) [Insane]](https://osu.ppy.sh/beatmapsets/111825#osu/290733)
+- DoubleTime
+  1. [azusa - Check my soul (Sumisola) [Insane]](https://osu.ppy.sh/beatmapsets/45664#osu/142462)
+  2. [Shoichiro Sakamoto - Eye of Aeon (KanaRin) [Insane]](https://osu.ppy.sh/beatmapsets/26528#osu/91139)
+  3. [nano - Now or Never (tutuhaha) [Now]](https://osu.ppy.sh/beatmapsets/51755#osu/158498)
+- ForceMod
+  1. [HitoshizukuP x Yama - Crazy nighT (Sephibro) [Crazy]](https://osu.ppy.sh/beatmapsets/109401#osu/285549)
+  2. [Kawada Mami - u/n (Reiji Maigo) [Insane]](https://osu.ppy.sh/beatmapsets/1142576#osu/2470326)
+- Tiebreaker
+  1. **[Uchu Conbini - Compass (Sing) [Space]](https://osu.ppy.sh/beatmapsets/1349881#osu/2794869)**
+
+### Seeding
+
+- NoMod
+  1. [KINEMA106 - Juusou (Mordred) [Yogurtt's Extra]](https://osu.ppy.sh/beatmapsets/1301393#osu/2709081)
+  2. [Felicity of the Millennium - Sacred Renaissance (rollpan) [NM2]](https://osu.ppy.sh/beatmapsets/1569544#osu/3205100)
+  3. [After the Rain - Shikioriori ni Tayutotte (\[ -Scarlet- \]) [LanJay rigged this. Ban Jesus too]](https://osu.ppy.sh/beatmapsets/837257#osu/1753120)
+  4. [Function Phantom - Variable (Chaoslitz) [Deluxe]](https://osu.ppy.sh/beatmapsets/751774#osu/1582594)
+- Hidden
+  1. [bj.HaLo - Ende (galvenize) [Another]](https://osu.ppy.sh/beatmapsets/44035#osu/148716)
+  2. [kors k - SigSig (AJT) [Nisan's Insane]](https://osu.ppy.sh/beatmapsets/1615650#osu/3298595)
+- HardRock
+  1. [THE ORAL CIGARETTES - 5150 (Kagetsu) [Collab Extra]](https://osu.ppy.sh/beatmapsets/705655#osu/1532288)
+  2. [HyuN - Guitian (ScubDomino) [Insane]](https://osu.ppy.sh/beatmapsets/891348#osu/1863256)
+- DoubleTime
+  1. [senya - Koyoi Index (jonathanlfj) [Lunatic]](https://osu.ppy.sh/beatmapsets/234455#osu/543814)
+  2. [Tsunamaru - Gin no Fluch (HelloSCV) [Insane]](https://osu.ppy.sh/beatmapsets/85721#osu/235309)
+
+## Match results
+
+### Grand Finals
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **[Chzaron](https://osu.ppy.sh/users/11607790)** ![][flag_US] | **6** | 1 | ![][flag_US] [Kenny](https://osu.ppy.sh/users/1225459) | [#1](https://osu.ppy.sh/community/matches/101038923) |
+
+### Finals
+
+#### Winners bracket
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **[Chzaron](https://osu.ppy.sh/users/11607790)** ![][flag_US] | **5** | 2 | ![][flag_US] [-ether-](https://osu.ppy.sh/users/6131258) | [#1](https://osu.ppy.sh/community/matches/101037695) |
+
+#### Losers bracket
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| [-ether-](https://osu.ppy.sh/users/6131258) ![][flag_US] | -1 | **0** | ![][flag_US] **[Kenny](https://osu.ppy.sh/users/1225459)** | *Forfeit* |
```

Comment:
```suggestion
| [-ether-](https://osu.ppy.sh/users/6131258) ![][flag_US] | -1 | **0** | ![][flag_US] **[Kenny](https://osu.ppy.sh/users/1225459)** | *forfeit* |
```

Reply:
why not sentence case

Reply:
because [tournament template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#stage-1) and convention

Reply:
i feel like neither should matter if it just looks dumb but alright.